core/src/block.rs

```diff
@@ -28,8 +32,21 @@ use crate::{
     wsv::WorldTrait,
 };
 
-/// Collection of actions, that is not a [`TriggerSet`]
-pub type Triggers = Vec<Action>;
+/// Recommendations for different types of triggers
+#[derive(Default, Debug, Clone, Decode, Encode, IntoSchema)]
+pub struct TriggerRecommendations {
+    /// Event based trigger recommendations. Should be executed before transactions.
+    pub event_triggers: Vec<EventAction>,
+    /// Time based trigger recommendations. Should be executed after transactions.
+    pub time_triggers: Vec<TimeAction>,
+}
+
+impl TriggerRecommendations {
```

Comment:
Why? Default isn't that much longer...

Reply:
I'm trying to follow this [rule](https://rust-lang.github.io/api-guidelines/predictability.html?highlight=default#constructors-are-static-inherent-methods-c-ctor):

> Note that it is common and expected for types to implement both `Default` and a `new` constructor. For types that have both, they should have the same behavior. Either one may be implemented in terms of the other.